src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -0,0 +1,76 @@
+import Cookies from 'universal-cookie';
+import jwtDecode from 'jwt-decode';
+
+// Apply the auth-related properties and functions to the Axios API client.
+export default function applyAuthInterface(authenticatedAPIClient, authConfig) {
+  /* eslint-disable no-param-reassign */
+  authenticatedAPIClient.appBaseUrl = authConfig.appBaseUrl;
+  authenticatedAPIClient.authServiceUrl = authConfig.authServiceUrl;
+  authenticatedAPIClient.accessTokenCookieName = authConfig.accessTokenCookieName;
+  authenticatedAPIClient.csrfCookieName = authConfig.csrfCookieName;
+  authenticatedAPIClient.loginUrl = `${authConfig.authServiceUrl}/login`;
+  authenticatedAPIClient.logoutUrl = `${authConfig.authServiceUrl}/logout`;
+  authenticatedAPIClient.refreshAccessTokenEndpoint = `${authConfig.authServiceUrl}/user_api/v1/account/refresh_access_token/`;
```

Comment:
Yes. I was trying to minimize the number of variables that need to be passed into the consuming app's build, but I agree it would be better for the full paths to be configurable. I will update.